phys/module_mp_thompson.F

```diff
@@ -464,7 +464,8 @@ SUBROUTINE thompson_init(hgt, orho, nwfa2d, nwfa, nifa,           &
             niCCN3 = -1.0*ALOG(naCCN1/naCCN0)/h_01
             nwfa(i,1,j) = naCCN1+naCCN0*exp(-((hgt(i,2,j)-hgt(i,1,j))/1000.)*niCCN3)
             airmass = 1./orho(i,1,j) * (hgt(i,2,j)-hgt(i,1,j))*dx*dy      ! kg
```

Comment:
airmass no longer needed

Reply:
Jimy,
Many thanks for the comments.  I will modify the codes accordingly.
Ming